src/Backend/Modules/Location/Domain/Location.php

```diff
@@ -0,0 +1,330 @@
+     */
+    private $showInOverview;
+
+    /**
+     * @var Collection
+     *
+     * @ORM\OneToMany(
+     *     targetEntity="Backend\Modules\Location\Domain\LocationSetting",
+     *     mappedBy="location",
+     *     cascade={"persist"},
+     *     orphanRemoval=true
+     * )
+     */
+    private $settings;
+
+    /**
+     * @var DateTime
+     *
+     * @ORM\Column(type="datetime")
+     */
+    private $createdOn;
+
+    /**
+     * @var DateTime
+     *
+     * @ORM\Column(type="datetime")
+     */
+    private $editedOn;
+
+    public function __construct(
+        Locale $locale,
+        string $title,
+        string $street,
+        string $number,
+        string $zip,
+        string $city,
+        string $country,
+        float $latitude,
+        float $longitude,
+        bool $showInOverview = true,
+        ?int $extraId = null
+    ) {
+        $this->locale = $locale;
+        $this->title = $title;
+        $this->street = $street;
+        $this->number = $number;
+        $this->zip = $zip;
+        $this->city = $city;
+        $this->country = $country;
+        $this->latitude = $latitude;
+        $this->longitude = $longitude;
+        $this->showInOverview = $showInOverview;
+        $this->extraId = $extraId;
+
+        $this->settings = new ArrayCollection();
+    }
+
+    public function update(
+        string $title,
+        string $street,
+        string $number,
+        string $zip,
+        string $city,
+        string $country,
+        float $latitude,
+        float $longitude,
+        bool $showInOverview = true
+    ) {
+        $this->title = $title;
+        $this->street = $street;
+        $this->number = $number;
+        $this->zip = $zip;
+        $this->city = $city;
+        $this->country = $country;
+        $this->latitude = $latitude;
+        $this->longitude = $longitude;
+        $this->showInOverview = $showInOverview;
+    }
+
+    public function getId(): int
+    {
+        return $this->id;
+    }
+
+    public function getLocale(): Locale
+    {
+        return $this->locale;
+    }
+
+    public function getExtraId(): int
+    {
+        return $this->extraId;
+    }
+
+    public function getTitle(): string
+    {
+        return $this->title;
+    }
+
+    public function getStreet(): string
+    {
+        return $this->street;
+    }
+
+    public function getNumber(): string
+    {
+        return $this->number;
+    }
+
+    public function getZip(): string
+    {
+        return $this->zip;
+    }
+
+    public function getCity(): string
+    {
+        return $this->city;
+    }
+
+    public function getCountry(): string
+    {
+        return $this->country;
+    }
+
+    public function getLatitude(): float
+    {
+        return $this->latitude;
+    }
+
+    public function getLongitude(): float
+    {
+        return $this->longitude;
+    }
+
+    public function isShowInOverview(): ?bool
```

Comment:
both in the constructor and the update the showInOverview can only be a boolean, so I don't think we need a ? here

Reply:
You are correct 1b5b3c46b